setup.py

```diff
@@ -19,13 +19,15 @@ def package_files(directory):
 # to include in the wheel, e.g. "../mlflow/server/js/build/index.html"
 js_files = package_files('mlflow/server/js/build')
 sagmaker_server_files = package_files("mlflow/sagemaker/container")
+alembic_files = ["../mlflow/alembic.ini"]
```

Comment:
Include alembic config in Python wheel - otherwise the `upgradedb` CLI doesn't work